projects/demo/src/modules/components/input-files/input-files.template.html

```diff
@@ -57,8 +57,7 @@
             <tui-input-files
                 [formControl]="control"
                 [accept]="accept"
-                [link]="link"
-                [label]="label"
+                [content]="content"
```

Comment:
add schematics, please

Reply:
Fixed, @splincode thanks for your checks!